ctapipe/containers.py

```diff
@@ -71,6 +71,11 @@
 UNKNOWN_ID = np.uint64(np.iinfo(np.uint64).max)
 
 
+obs_id_field = partial(Field, UNKNOWN_ID, description="Observation Block ID")
+event_id_field = partial(Field, UNKNOWN_ID, description="Array Event ID")
+tel_id_field = partial(Field, UNKNOWN_ID, description="Telescope ID")
```

Comment:
According to the data model, tel ids are uint16, not uint64 I think. Let me check.

Reply:
yes, they are - I missed that.  Since we don't expect even more than a few hundred values there, smaller storage saves some space.

Reply:
`obs_ids` could even be  32 bit for that matter if we really want to save some space - here we do store them per event, so it could be useful

Reply:
The DL0 data model has them as uint64

Reply:
I only changed it for `tel_id` for now.
Should I change `UNKNOWN_ID` to uint32 and introduce `UNKNOWN_EVENT_ID` as the uint64 default? `UNKNOWN_ID` is also used for the scheduling block id, but this wouldn't be a problem, would it?

Reply:
But you are probably correct, ...

```
In [9]: (np.iinfo(np.uint32).max * 1 * u.min).to(u.year)
Out[9]: <Quantity 8165.95804814 yr>
```

Reply:
Let's do what the data model does and ask Igor to consider changing it there